packages/core/src/fhir-to-cda/cda-templates/components/observations.ts

```diff
@@ -0,0 +1,57 @@
+import { Observation } from "@medplum/fhirtypes";
+import { CDAInstanceIdentifier, CDACodeCV, Entry } from "../types";
+import {
+  buildInstanceIdentifiersFromIdentifier,
+  buildCodeCVFromCodeableConcept,
+  buildInstanceIdentifier,
+  withoutNullFlavorString,
+} from "../commons";
+import { classCodeAttribute, moodCodeAttribute } from "../constants";
+
+export interface CDAObservation {
```

Comment:
move to the types file? TBH, I like individual files instead of a huge one, but also I don't think that types should live under "templates"